homeassistant/components/script/blueprints/confirmable_notification.yaml

```diff
@@ -0,0 +1,74 @@
+blueprint:
+  name: Confirmable Notification
+  description: >-
+    A script that sends an actionable notification with a confirmation before
+    running the specified action.
+  domain: script
+  source_url: https://github.com/home-assistant/core/blob/dev/homeassistant/components/script/blueprints/confirmable_notification.yaml
```

Comment:
We should point this at the master branch. If we ever allow updating, we wouldn't want people to update to a dev version that uses things that are not available in master.

Reply:
Adjusted